src/GraphQL/Types/Scalars/IdGraphType.cs

```diff
@@ -2,22 +2,32 @@
 
 namespace GraphQL.Types
 {
+    /// <summary>
+    /// The ID scalar graph type represents a string identifier, not intended to be human-readable.
+    /// When accepted as an input type, any string or integer input value will be accepted as an ID.
+    /// </summary>
     public class IdGraphType : ScalarGraphType
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="IdGraphType"/> class.
+        /// </summary>
         public IdGraphType()
         {
             Name = "ID";
             //Description =
             //    "The `ID` scalar type represents a unique identifier, often used to re-fetch an object or " +
             //    "as key for a cache. The `ID` type appears in a JSON response as a `String`; however, it " +
-            //    "is not intended to be human-readable. When expected as an input type, any string (such " +
+            //    "is not intended to be human-readable. When accepted as an input type, any string (such " +
```

Comment:
This wording is from spec:
![изображение](https://user-images.githubusercontent.com/21261007/102387671-b7df7a80-3fe1-11eb-8b04-59cf3dca625a.png)

Reply:
Well, it doesn't look right to me.  Perhaps because the context is different (in the code, the context is the field, whereas in the spec, the context is "input coercion").  Perhaps it's fine.  In any case I'll just leave it.  It's not like it matters.

Reply:
```suggestion
            //    "is not intended to be human-readable. When expected as an input type, any string (such " +
```